requirements.txt

```diff
@@ -0,0 +1,4 @@
+pandas
+gradio
+sklearn
+joblib
```

Comment:
versions should be strict. e.g. pandas==x.y.z
where x is the major version, y is minor, z is patch version.